docs/spec_draft.md

```diff
@@ -127,3 +127,79 @@ specification.
 %z = stablehlo.add %x, %y : tensor<2x2xf32>
 // %z: [[6, 8], [10, 12]]
 ```
+
+## stablehlo.maximum
+
+`stablehlo.maximum(lhs, rhs) -> result`
+
+### Semantics
+
+Performs element-wise max operation on tensors `lhs` and `rhs` and produces a
+`result` tensor. For Floating-point element type, implements IEEE 754 semantics:
+Returns the larger of two arguments, propagating `NaN`s and treating `-0` as
+less than `+0`.
+
+### Arguments
+
+| Name | Type |
+|-|-|
+| `lhs` | tensor of Integer, Floating-point element types |
```

Comment:
After reading the spec draft some more, I feel ambivalent about capitalization of "Integer types". Are these names special enough to be capitalized? For comparison, https://llvm.org/docs/LangRef.html doesn't capitalize them.

Reply:
done.